frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -1155,6 +1155,13 @@ func dispatchCopy(d *dispatchState, cfg copyConfig) error {
 		commitMessage.WriteString("COPY")
 	}
 
+	if cfg.chown != "" {
+		commitMessage.WriteString(" " + "--chown=" + cfg.chown)
+	}
+	if cfg.chmod != "" {
+		commitMessage.WriteString(" " + "--chmod=" + cfg.chmod)
+	}
+
```

Comment:
--parents: is reasonable
--from: I think it's not necessary (because images only includes the last stage, so adding from doesn't help)
--checksum: Checksum is usually quite long. So I think including it will make the history look more confusing?